src/lms7002m/mcu_dc_iq_calibration.cpp

```diff
@@ -1,6 +1,7 @@
 #include "mcu_programs.h"
 
-LIME_API const uint8_t mcu_program_lms7_dc_iq_calibration_bin[16384] = {
+// LIME_API // Is this really needed here?  It triggers some ugly warnings.
```

Comment:
The macro should just be setting gcc visibility. In this case the pointer to the mcu program is used outside of the main library for the GUI. Whats the warning?